src/Liveness.zig

```diff
@@ -621,11 +621,18 @@ pub fn categorizeOperand(
                 if (inst_data.operand == operand_ref and operandDies(l, body[0], 0))
                     return .tomb;
 
-                if (cond_extra.data.then_body_len != 1 or cond_extra.data.else_body_len != 1)
+                if (cond_extra.data.then_body_len > 2 or cond_extra.data.else_body_len > 2)
+                    return .complex;
+
+                const then_body = air.extra[cond_extra.end..][0..cond_extra.data.then_body_len];
+                const else_body = air.extra[cond_extra.end..][cond_extra.data.then_body_len .. cond_extra.data.then_body_len + cond_extra.data.else_body_len];
```

Comment:
Not a blocker, but this would be neater as `air.extra[cond_extra.end + cond_extra.data.then_body_len ..][0..cond_extra.data.else_body_len]`